tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/api/TestIO.java

```diff
@@ -177,8 +176,6 @@ public void testCopyInWithDevice() {
         FloatArray arrayB = createAndInitializeArray(N);
         FloatArray arrayC = new FloatArray(N);
 
-        TornadoRuntime.getTornadoRuntime().getDefaultDevice().reset();
```

Comment:
Any reason of this deletion? Is it deliberate?

Reply:
There is no need for this call in this test